frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -0,0 +1,284 @@
+<script lang="ts">
+	import Paper from "@smui/paper";
+	import { fade } from "svelte/transition";
+	import { Label } from "@smui/common";
+	import Chip from "@smui/chips";
+	import { Svg } from "@smui/common";
+	import { showNewSlice, slices, sliceToEdit } from "../../stores";
+	import IconButton, { Icon } from "@smui/icon-button";
+	import Select, { Option } from "@smui/select";
+	import { clickOutside } from "../../util/clickOutside";
+	import { mdiPlus, mdiClose } from "@mdi/js";
+	import { showSliceFinder } from "../../stores";
+	import { model, metric } from "../../stores";
+	import Button from "@smui/button";
+	import { getMetricsForSlices } from "../../api/slice";
+	import { ZenoService, type MetricKey } from "../../zenoservice";
+
+	let slice_data = [];
+	// dummy data for presenting frontend UI
+	let minimumSizes = ["2", "3", "4", "5", "6", "7"];
+	let minimumSize = "5";
+	let max_ls = ["4", "5", "6", "7", "8"];
+	let max_l = "5";
+	let sliceFinderKeys = ["data"];
```

Comment:
I think this is wrong. This should list the columns you can pick to do slice finding across.

Reply:
e.g. "brightness", "red count" etc.